### ISSUE TITLE
Transform should have easy way to create S3 and local file systems

### ISSUE THREAD

**Issue by adamvinueza** [adamvinueza/transformer#32]
Something like this:
```
from transformer.transform import Transform, S3FS, LocalFS
aws_profile='dev'
tr = Transform(src_fs=S3FS(aws_profile), dest_fs=LocalFS())
tr.copy(src, dest)
```
The S3FS function should automatically find credentials given the specified AWS profile name.

**Comment by adamvinueza** [comment]
Maybe a different namespace?
```
from transformer.transform import Transform
from transformer.fs import S3FS, LocalFS
aws_profile='dev'
tr = Transform(src_fs=S3FS(aws_profile), dest_fs=LocalFS())
tr.copy(src, dest)
```

**Comment by adamvinueza** [comment]
Fixed in #33.
